Update README.md

marlenaweidenauer opened the pull request:
added preliminary Matlab client

marlenaweidenauer commented:
added modified Matlab client and corresponding documentation

linusseelinger commented:
Seems like you currently have two clients in clients/matlab. Only one of them is executed in automated testing via testing/clients/TestMatlab.m, so I believe one of them should be removed. Apart from that, looks good!

marlenaweidenauer commented:
I changed the file matlab_client.m. It uses a posterior model now, with configuration as well (should this be tested too?). So I think one can delete the matlabClient.m file and replace it with this one.
I changed the corresponding README aswell.

dolgov commented:
You can add your branch to .github/workflows/client-matlab.yml to invoke the CI tests on every push of your branch, in addition to main. You can then test your edits without pull requests. In particular, it looks like either matlab_client.m should be renamed, or TestMatlab.m should be updated to run the new file.

linusseelinger commented:
Looks good, thanks a lot! :)
